src/ucp/core/ucp_ep.c

```diff
@@ -1933,17 +1933,20 @@ int ucp_ep_config_get_multi_lane_prio(const ucp_lane_index_t *lanes,
     return -1;
 }
 
-static void ucp_ep_config_cm_lane_info_str(ucp_worker_h worker,
-                                           const ucp_ep_config_key_t *key,
-                                           ucp_lane_index_t lane,
-                                           char *buf, size_t max)
+void ucp_ep_config_cm_lane_info_str(ucp_worker_h worker,
+                                    const ucp_ep_config_key_t *key,
+                                    ucp_lane_index_t lane,
+                                    ucp_rsc_index_t cm_index,
+                                    char *buf, size_t max)
 {
-    ucp_context_h context        = worker->context;
-    ucp_rsc_index_t cmpt_index   = worker->cms[lane].cmpt_idx;
-    const ucp_tl_cmpt_t *tl_cmpt = &context->tl_cmpts[cmpt_index];
-
-    ucs_snprintf_zero(buf, max, "lane[%d]: %2d:%s cm",
-                      lane, cmpt_index, tl_cmpt->attr.name);
+    ucp_rsc_index_t cmpt_index = (cm_index != UCP_NULL_RESOURCE) ?
+                                 worker->cms[cm_index].cmpt_idx :
+                                 UCP_NULL_RESOURCE;
+    char cm_name[UCT_TL_NAME_MAX];
+
+    ucs_snprintf_zero(buf, max, "lane[%d]: %2d:%s cm [%d]", lane, cmpt_index,
```

Comment:
minor: maybe write 'cmpt_index' in the print? otherwise it's not clear what the number before the cm_name means.

Reply:
I'm using the same format as in `ucp_ep_config_lane_info_str()` when printing TL resources, but instead of `rsc_index`, it prints `cmpt_index`

Reply:
done